### ISSUE TITLE
[android] Network Security Configuration allows cleartext traffic for all domains

### ISSUE THREAD

**Issue by rtsisyk** [organicmaps/organicmaps#6642]
Your app's Network Security Configuration allows cleartext traffic for **all** domains. This could allow eavesdroppers to intercept data sent by your app. If that data is sensitive or user-identifiable, it could impact upon the privacy of your users.

Consider only permitting encrypted traffic by setting the [cleartextTrafficPermitted](https://developer.android.com/training/articles/security-config#CleartextTrafficPermitted) flag to false, or adding an encrypted policy for specific domains.

---------

android/app/src/main/res/xml:

```
<network-security-config>
  <base-config cleartextTrafficPermitted="true">
  </base-config>
  <domain-config cleartextTrafficPermitted="true">
     <domain includeSubdomains="true">organicmaps.app</domain>
     <domain includeSubdomains="true">omaps.app</domain>
  </domain-config>
</network-security-config>
```

`<base-config cleartextTrafficPermitted="true">` should be removed.

**Comment by biodranik** [comment]
Yes, it was done to fix many user reports when images in bookmark descriptions were not loaded over HTTP. To reproduce the issue, create HTML bookmark description with some external HTTP links to images, music, videos, etc. It won't work if clear text is disabled.
